src/frontend/js/components/AshleyEditor/index.tsx

```diff
@@ -22,6 +22,7 @@ interface MyEditorProps {
   placeholder?: string;
   target: HTMLInputElement;
   emojiConfig?: EmojiPluginConfig;
+  linkPlaceholder?: string;
```

Comment:
Is this props really optional ? The props are editable and can be changed/removed ?

Reply:
Sorry for the delay.
I made this props optional because the draftjs plugin provides a default placeholder.
One the editor is initialized, this cannot be changed/removed.